src/helper-modules/companies-house/wrap-companies-house-http-error-callback.ts

```diff
@@ -9,21 +9,25 @@ type CompaniesHouseHttpErrorCallback = (error: Error) => ObservableInput<never>;
 export const createWrapCompaniesHouseHttpGetErrorCallback =
   ({ messageForUnknownError, knownErrors }: { messageForUnknownError: string; knownErrors: KnownErrors }): CompaniesHouseHttpErrorCallback =>
   (error: Error) => {
-    if (error instanceof AxiosError && error.response && typeof error.response.data === 'object') {
-      const errorResponseData = error.response.data;
-      let errorMessage: string;
+    if (error instanceof AxiosError) {
```

Comment:
could we guard clause this please to remove some complexity (`if (!isAxiosError(error)... bla` -- Also by doing `isAxioxError` it helps with typescript inferring the type better -- should make it complain less about accessing `error.response` etc

Reply:
Have added `&& error?.response` as `404` now returns empty `error.response.data` object.